src/common/imageio_rawspeed.cc

```diff
@@ -378,6 +378,9 @@ dt_imageio_retval_t dt_imageio_open_rawspeed_sraw(dt_image_t *img, RawImage r, d
   // if buf is NULL, we quit the fct here
   if(!mbuf) return DT_IMAGEIO_OK;
 
+  // We test for monochrome before allocating the mipmap cache 
+  if(cpp == 1) img->flags |= DT_IMAGE_MONOCHROME;
```

Comment:
Have you tested raw loading from a leica monochrom raw? There are samples at: https://raw.pixls.us/ if you search for "monochrom"

Reply:
Yes, i tested  both  leica monochrom DNGs i found there.

Aah i see @LebedevRI commenting

> And same for the other modules that are between rawprepare and demosaic AND require CFA: invert, cacorrect.